clinicadl/data/datatype/raw/custom.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from clinicadl.data.datatype.file_type import FileType
+from clinicadl.data.datatype.modalities import Custom
+
+from .base import RawData
+
+
+class RawCustom(RawData, Custom):
+    """
+    Configuration class for raw custom imaging data with a user-defined suffix.
+
+    This class represents custom images in their raw BIDS format and provides
+    methods to define file patterns and descriptions.
+    """
+
+    def _get_bids_filetype(self, reconstruction: Optional[str] = None) -> FileType:
```

Comment:
```suggestion
    def _get_bids_filetype(self) -> FileType:
```
Can we get rid of `reconstruction` when we don't use it? Or is it here on purpose?